### PR TITLE
Frame switches

### PR DESCRIPTION
**Pull request by aryangupta701**
added feature to generate script corresponding to each frame switch

### CODE REVIEW

**Comment by psiinon** [comment]
Looks like its failing the lint check

**Comment by aryangupta701** [comment]
Leave this one I will fix this later

**Comment by aryangupta701** [comment]
I want to merge this PR first: https://github.com/zaproxy/browser-extension/pull/55

**Comment by aryangupta701** [comment]
will fix it asap

File: test/ContentScript/webpages/testFrame.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<body>
+    <button id="test-btn">Test ME</button>
+    <script>
+        const button = document.getElementById('test-btn'); 
+        button.addEventListener('click', ()=>{
+            button.after("Clicked !!!"); 
+        })
+    </script>
+</body>
+</html>`
```
Comment (thc202) [inline test/ContentScript/webpages/testFrame.html:15]:
```suggestion
</html>
```

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

File: source/ContentScript/recorder.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ import {
   ZestStatementElementClick,
   ZestStatementElementSendKeys,
   ZestStatementLaunchBrowser,
+  ZestStatementSwichToFrame,
```
Comment (ricekot) [inline source/ContentScript/recorder.ts:27]: There's a typo in "Switch".

Comment (thc202) [reply source/ContentScript/recorder.ts:27]: Nice catch (the name of the element is correct).

**Pull request merged**